### PR TITLE
Update Russian language

### PR DESCRIPTION
**Pull request by WebVPF**

### CODE REVIEW

File: modules/system/lang/ru/lang.php [hunk 1]
```diff
@@ -95,6 +140,9 @@
         'refresh_success'         => 'Выбранные плагины успешно обновлены.',
         'remove_confirm'          => 'Вы уверены, что хотите удалить выбранные Плагины? Это также приведет к удалению всех связанных данных.',
         'remove_success'          => 'Выбранные плагины успешно удалены.',
+        'replace' => [
+            'multi_install_error' => 'Замена нескольких плагинов в настоящее время не поддерживается'
+        ]
```
Comment (bennothommo) [inline modules/system/lang/ru/lang.php]:
```suggestion
            'multi_install_error' => 'Замена нескольких плагинов в настоящее время не поддерживается',
        ],
```

**Comment by bennothommo** [comment]
Thanks @WebVPF! :)

**Comment by WebVPF** [comment]
@LukeTowers @bennothommo I want to warn you.
Today someone said that you can install paid October plugins in Winter CMS.
They say it's a violation of the license.
Threatened with legal action.
In the Russian-speaking community of October, they are asked not to raise a fuss for the time being.

I will delete this comment after you read it.

**Comment by LukeTowers** [comment]
Interesting, sounds like a bug in the API code on our side, installing paid plugins should work, but only if you have a valid project ID for the product as determined by the October marketplace. Shoot me an email or message me on discord with the details and I'll look into it and fix it if it's an issue on our side. Thanks for the report @WebVPF!

**Comment by WebVPF** [comment]
@LukeTowers I didn't install paid plugins. I'm just reporting that this was reported to Sam and Bobkov. They threaten to sue. This is written by Lavtsevich (Lovata) and Biz-Mark

**Comment by WebVPF** [comment]
@LukeTowers https://t.me/octobercms_rus/202867

**Pull request merged**